USER_MANUAL.md

```diff
@@ -385,5 +385,12 @@ Manual steps:<br>
   1. Start the gym again and note that the level actors should be at their previous shutdown positions, not their original positions. You can do this repeatedly.
   1. The test has now passed. Don’t forget to revert the two settings changes you made before you run another test.
 
+##### Cross-Server possession gym
+* Functional tests the Controller remote possess the Pawn
+* How to test:
+  * Set the current map to the map which in the folder(/Game/Content/Maps/FunctionalTests/)  
```

Comment:
You've added 3 maps to this PR, but only added one use-case. 
If all that's required is selecting the map in the functional test framework, and running the test, there's no need to add an entry to the user_manual.